pkg/tfvars/ibmcloud/ibmcloud.go

```diff
@@ -2,14 +2,21 @@ package ibmcloud
 
 import (
 	"encoding/json"
+	"fmt"
+	"strings"
 
 	"github.com/pkg/errors"
 
+	configv1 "github.com/openshift/api/config/v1"
 	"github.com/openshift/installer/pkg/tfvars/internal/cache"
 	"github.com/openshift/installer/pkg/types"
+	ibmcloudtypes "github.com/openshift/installer/pkg/types/ibmcloud"
 	ibmcloudprovider "github.com/openshift/machine-api-provider-ibmcloud/pkg/apis/ibmcloudprovider/v1"
 )
```

Comment:
```suggestion
	configv1 "github.com/openshift/api/config/v1"
	
	"github.com/openshift/installer/pkg/tfvars/internal/cache"
	"github.com/openshift/installer/pkg/types"
	ibmcloudtypes "github.com/openshift/installer/pkg/types/ibmcloud"
	ibmcloudprovider "github.com/openshift/machine-api-provider-ibmcloud/pkg/apis/ibmcloudprovider/v1"
)
```

Reply:
`gofmt` preferred this as is, so I've changed it back
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_installer/7632/pull-ci-openshift-installer-master-gofmt/1721621569781895168

Reply:
makes sense, sorry about that.

Reply:
No worries, I always have trouble with them without `gofmt`'s help.